src/AllBookings/BookingCard/__test__/DateAndPassenger.test.js

```diff
@@ -16,8 +16,15 @@ const departure = {
 
 const status = bookingStatus['CONFIRMED'];
 
+const bookingDate = '2017-11-23T20:40:00.000Z';
```

Comment:
I would suggest removing these two variables and pass values directly to the component - or at least remove the empty line on line 18, my OCD meter is broken :D